source/agora/test/Base.d

```diff
@@ -717,7 +717,7 @@ public class TestAPIManager
         Duration timeout = 10.seconds,
         string file = __FILE__, int line = __LINE__)
     {
-        this.expectHeightAndPreImg(iota(GenesisValidators), height, enroll_header,
+        this.expectHeightAndPreImg(iota(this.test_conf.node.limit_test_validators), height, enroll_header,
```

Comment:
Should we make `GenesisValidator` have the value of `limit_test_validators` even though it does not need to be done now?

Reply:
I also thought about that but the problem is that `GenesisValidators` is a public import in `base.d` used in many tests that do not have any `testConf`.